stock_barcodes/static/src/barcodes_models_utils.esm.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module */
+/* Copyright 2022 Tecnativa - Alexandre D. Díaz
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+const barcodeModels = [
+    "stock.barcodes.action",
+    "stock.picking",
+    "stock.picking.type",
+    "wiz.candidate.picking",
+    "wiz.stock.barcodes.new.lot",
+    "wiz.stock.barcodes.read",
+    "wiz.stock.barcodes.read.inventory",
+    "wiz.stock.barcodes.read.picking",
+    "wiz.stock.barcodes.read.todo",
+];
+
+export function isAllowedBarcodeModel(modelName) {
+    return barcodeModels.indexOf(modelName) !== -1;
```

Comment:
A little bit more expressive:
```suggestion
    return barcodeModels.includes(modelName);

Reply:
Thank you ! Changed